app/validation/validator.py

```diff
@@ -151,6 +153,75 @@ def validate_range_types_from_answers(self, json_to_validate):
 
         return errors
 
+    def _validate_min_max_ranges(self, json_to_validate):
+        errors = []
+        answers = {}
+
+        for block in self._get_blocks(json_to_validate):
+            for answer in self._get_answers_for_block(block):
+                answers[answer.get('id')] = answer
+
+        for answer_id, answer in answers.items():
+            referenced_answer_id = self._get_referenced_answer_id(answer)
+
+            if referenced_answer_id:
+                answer_min_max = self._get_range_for_answer(answer, answers)
+                # referenced_answer_range = self._get_range_for_answer(answers[referenced_answer_id], answers)
+                answer_range = answer_min_max[1] - answer_min_max[0]
```

Comment:
We could assign multiple variables from the `_get_range_for_answer` call:
```
minimum, maximum = self._get_range_for_answer(answer, answers)
```
so we can remove the need for using the `0` and `1` array accesses